untidy/contaminators.py

```diff
@@ -25,5 +125,63 @@ def contaminate_strings_with_noise(a, corruption_level=4):
 
 
 """ Functions to contaminate numerical columns: """
+def add_outliers(clean_data, corruption_level = 4):
+    """
+    Contaminate data with obvious outliers
+
+    Parameters
+    ----------
+    clean_data: pd.Series or pd.DataFrame
+        data to be contaminated with missing values
+    corruption_level: int, optional
+
+    Returns
+    -------
+    contaminated data as pd.DataFrame
+
+    """
+
+    data_raw = clean_data.copy()
+    data = data_raw.copy()
+
+    # get numerical columns
+    if isinstance(data, pd.DataFrame):
+        num_cols = list(data.select_dtypes(include = np.number).columns)
+        data = data[num_cols]
+        num_limits = {col:(data[col].min(),data[col].max()) for col in num_cols}
+    else:
+        num_limits = (data.min(), data.max())
+
+    # get data shape
+    row_idxs, col_idxs = data_check(data)
+
+    nan_idxs = random_indices(row_idxs, col_idxs, corruption_level = corruption_level)
+ 
+    # random sign function
+    coin_flip = lambda: 1 if random.random() > 0.5 else -1
+    
+    # contaminate data
+    for x, y in nan_idxs:
+
+        coin = coin_flip()
+
+        if coin>0:
+            # +/- 100 times the max
+            if isinstance(data, pd.DataFrame):
+                data.iloc[x:x+1, y:y+1] = coin_flip() * num_limits[num_cols[y]][1] * 100
+            else:
+                data[x] = coin_flip() * num_limits[1] * 100
+        else:
+            # +/- 100 times the min
+            if isinstance(data, pd.DataFrame):
+                data.iloc[x:x+1, y:y+1] = coin_flip() * num_limits[num_cols[y]][0] * 100
+            else:
+                data[x] = coin_flip() * num_limits[0] * 100
 
-""" Functions to contaminate string columns: """
+    # add contamination to original dataframe
+    if isinstance(data, pd.DataFrame):
+        data_raw[num_cols] = data[num_cols]
```

Comment:
Shouldn't we return a new dataframe instead of `data_raw` ? Like `data_contaminated` or just `data`